html_app/instructor.js

```diff
@@ -0,0 +1,65 @@
+
+$(function(){
+
+    $('.scb_s_dashboard_table_row').hover(function(){
+	    $('.scb_s_dashboard_link', this).toggle();
+    });
+    /**
+     * Course setup
+     */
+    $('.scb_f_course_setup_create_new_course_option input').click(function(){
```

Comment:
make a css class to add the same click handler to both, do `$(".scb_f_use_existing_course").toggle()` and a toggle for the other class, and a conditional for the prop.  Admittedly that isn't saving you a ton here...